addons/purchase/purchase.py

```diff
@@ -29,13 +29,10 @@ def _amount_all(self):
                 'amount_total': amount_untaxed + amount_tax,
             })
 
-    @api.multi
-    def _inverse_date_planned(self):
-        for order in self:
-            order.order_line.write({'date_planned': self.date_planned})
-
     @api.depends('order_line.date_planned')
     def _compute_date_planned(self):
+        # There is no inverse function on purpose, to avoid resetting the scheduled date on the
```

Comment:
instead of "avoid resetting", put something like "it may be different on each line". also maybe put the comment next to the field instead of this method